omod/src/main/resources/config.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<module configVersion="1.2">
+
+	<id>bahmni-events</id>
+	<name>Bahmni Events</name>
+	<version>1.0.0-SNAPSHOT</version>
+	<package>${MODULE_PACKAGE}</package>
+	<author>Bahmni TW</author>
```

Comment:
Now sure if we should say author as "TW". Maybe just Bahmni is fine.

Reply:
Removed TW